docs/platforms/react-native/data-management/data-collected.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Data Collected
+description: "See what data is collected by the Sentry SDK."
+sidebar_order: 1
+---
+
+Sentry takes data privacy very seriously and has default settings in place that prioritize data safety, especially when it comes to personally identifiable information (PII) data. When you add the Sentry SDK to your application, you allow it to collect data and send it to Sentry during the runtime of your application.
+
+The category types and amount of data collected vary, depending on the integrations you've enabled in the Sentry SDK. This page lists data categories that the Sentry React Native SDK collects.
+
+Many of the categories listed here require you to set <PlatformLink to="/configuration/options/#send-default-pii">`sendDefaultPii: true`</PlatformLink> in your `Sentry.init({})` config.
+
+## HTTP Headers
+
+By default, the Sentry SDK attaches HTTP request information, such as URL, user-agent, referrer, and other headers, to the event. To disable this behavior, remove the `Sentry.httpContextIntegration` from the integrations array.
+
+Failed Fetch and XHR requests error events from `Sentry.httpClientIntegration` don't contain header or cookie data.
+
+To start sending headers and cookies, set <PlatformLink to="/configuration/options/#send-default-pii">`sendDefaultPii: true`</PlatformLink> in your `Sentry.init({})` config.
+
+## Cookies
+
+By default, the Sentry SDK doesn't send cookies. Sentry tries to remove any cookies that contain sensitive information, such as the Session ID and CSRF Token cookies.
+
+To start sending cookies, set <PlatformLink to="/configuration/options/#send-default-pii">`sendDefaultPii: true`</PlatformLink> in your `Sentry.init({})` config.
+
+## Information About Logged-in User
+
+By default, the Sentry SDK doesn't send any information about the logged-in user, such as email address, user ID, or username. Even if enabled, the type of logged-in user information you'll be able to send depends on the integrations you enable in Sentry's SDK. Most integrations won't send any user information. Some will only set the user ID, but there are a few that will set the user ID, username, and email address.
+
+To start sending logged-in user information, set <PlatformLink to="/configuration/options/#send-default-pii">`sendDefaultPii: true`</PlatformLink> in your `Sentry.init({})` config.
+
+## Users' IP Addresses
+
+By default, the user's IP address is inferred by the Sentry backend services based on the incoming request.
+
+To disable sending the user's IP address, override the default value by a custom String value.
```

Comment:
`h`: this should be guarded by `sendDefaultPii` as well like on [Python](https://docs.sentry.io/platforms/python/data-management/data-collected/#users-ip-address) - I'm assuming we need [this](https://github.com/getsentry/sentry-react-native/issues/4462) to be done first, which in turn requires a JS SDK bump, is that correct?

Reply:
Yes, until the upgrade to JS v9 the IP address is inferred, even without `{{auto}}`.

https://github.com/getsentry/team-sdks/issues/118#issuecomment-2602812892

Reply:
and v9 bump will be part of RN v7, right? so we should update this section with v7 release.

Reply:
Yes.